shopinvader/models/shopinvader_backend.py

```diff
@@ -5,16 +5,21 @@
 from contextlib import contextmanager
 
 from odoo import _, api, fields, models, tools
+from odoo.addons.http_routing.models.ir_http import slugify
 from odoo.addons.server_environment import serv_config
 from odoo.http import request
 
 
 class ShopinvaderBackend(models.Model):
     _name = "shopinvader.backend"
-    _inherit = "collection.base"
+    _inherit = ["collection.base", "server.env.mixin"]
     _description = "Shopinvader Backend"
 
     name = fields.Char(required=True)
+    tech_name = fields.Char(
```

Comment:
@simahawk Why do we need a tech_name to use server_env????? server_env?

Reply:
because ppl use "name" for what is meant to be: a human-readable name and might what to change it.

Reply:
IMO since the backend is a technical model, I'm comfortable to use its name for technical things... that creates a strong link. But both POV are right :smirk: